_source/lib/apollos.coffee

```diff
@@ -37,10 +37,11 @@ Apollos.user.login = {}
 ###
 Apollos.person = ->
 
-  userDoc = Apollos.user()
+  user = Apollos.user()
 
-  person = Apollos.people.findOne
-    userId: userDoc._id
+  if user.rock?.guid
```

Comment:
@jbaxleyiii we can totally do that. that's just where it was already being set during sync. i move?